### DISCUSSION TITLE
RaspAP on DietPi install script fails

### DISCUSSION THREAD

**Discussion by ingoiot** [RaspAP/raspap-webgui#1471] [Q&A] [answered]
hi,

does anyone have any idea why the installation script is failing?
i attach a pastebin of the install log.

https://pastebin.com/Z5fkRiKu
> RaspAP Install: Configure installation
> Detected OS: Debian GNU/Linux 12 (bookworm) 64-bit
> Using GitHub repository: RaspAP/raspap-webgui 3.0.2 branch
> Configuration directory: /etc/raspap
> lighttpd root: /var/www/html? [Y/n]:

**Comment by billz** [comment]
The lighttpd dir `/var/www/html` doesn't exist. Add it with `sudo mkdir /var/www/html` and run the installer again.

**Reply by ingoiot** [reply]
i did,
the installer renames it and stops the same.

```
> root@DietPi:~# ls -la /var/www/html
> ls: cannot access '/var/www/html': No such file or directory
> root@DietPi:~# ls -la /var/www/html.2023-12-02-15\:32/

```

**Reply by billz** [reply]
Hard to say. The installer is unable to move files to the webroot, which is created by rsync if it doesn't exist.
This isn't observed on clean installs. Suggest you check the DietPi issues and/or forums, eg.: https://github.com/MichaIng/DietPi/issues/2813

**Reply by billz** [reply, marked as answer]
Try executing `rsync --version`. It might not be included in the DietPi image. 
There's been no compatibility testing with DietPi, although RaspAP is reported to install on it. 
Best advice here is to follow the [manual install steps](https://docs.raspap.com/manual/).

**Reply by ingoiot** [reply]
In fact, rsync and apt-utils were missing.
It installs now, but hostapd fails to start.
Maybe I'll skip that for now.
It looks like the installation scripts are interfering with each other too much.
